draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2302,6 +2363,29 @@ the subsection that gives the corresponding parameters.
 | secp256k1                 | {{suites-secp256k1}} |
 | BLS12-381                 | {{suites-bls12381}}  |
 
+## Defining a new hash-to-curve suite {#new-suite}
+
+The RECOMMENDED way to define a new hash-to-curve suite is:
+
+1. E, F, p, and m are determined by the elliptic curve and the base field.
+
+2. Choose a sgn0 variant following the guidelines in {{sgn0-variants}}.
+
+3. Choose hash function H meeting the requirements in {{hashtobase-sec}},
+   and compute L as described in that section.
+
+4. Choose a mapping following the guidelines in {{choosing-mapping}},
+   and select any required parameters for that mapping.
+
+5. Choose h\_eff to be either the cofactor of E or, if a fast cofactor
+   clearing method is to be used, a value appropriate to that method
+   as discussed in {{cofactor-clearing}}.
+
+6. Choose a Suite ID following the guidelines in {{suiteIDformat}}.
+
+Note that when standardizing a new elliptic curve, corresponding hash-to-curve
```

Comment:
it is not common to standardize curves. Using curves, on the other hand, is more often the case.

Reply:
You're totally right that it's not common! That said, I think it's a *good* thing to recommend that people standardizing curves generate their own hash-to-curve suites. No? Is there a reason not to include this suggestion?

Reply:
(Also note: this suggestion was in prior versions of the document---I just moved it here because it seemed to be the right place for it.)

Reply:
My point is that the suggestion must be broader, not only when someone is stardardizing a curve. 
The suggestion could say something like :  "anyone using a curve that is not listed in this document should generate a hash to curve method following this recommendation"

Reply:
Ah, got it!